alf/experience_replayers/replay_buffer.py

```diff
@@ -270,10 +298,27 @@ def get_batch(self, batch_size, batch_length):
                         non-zero priorities in the buffer.
         """
         with alf.device(self._device):
+            recent_batch_size = 0
+            if self._recent_data_ratio > 0:
```

Comment:
add assert recent_data_ratio * batch_size / num_env < recent_data_steps.
Otherwise, there isn't enough recent data, and the configuration is wrong.